### PR TITLE
CONCF-806 | Additional data for getData method

### PR DESCRIPTION
**Pull request by vforge**
/cc @hakubo @bognix @rogatty

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/CuratedContent/CuratedContentController.class.php [hunk 1]
```diff
@@ -440,14 +440,24 @@ public function getData( ) {
...
-				// sections
-				if ( !empty( $section['title'] ) && empty( $section['featured'] ) ) {
+				// update information about node type
+				$section['node_type'] = 'section';
+
+				// rename $section['title'] to $section['label']
+				$section['label'] = $section['title'];
+				unset( $section['title'] );
+
+				if ( !empty( $section['label'] ) && empty( $section['featured'] ) ) {
```
Comment (bognix) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php:450]: I think it doesn't solve our problem.
We wanted to have distinction for sections inside "curated" block so we don't need to do multiple ifs in Mercury.
Here you add node type to all sections (featured and optional) so we still need to add this if.

Comment (vforge) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php:450]: After (offline) talk we disagreed with that comment.

**Comment by bognix** [comment]
LGTM

File: extensions/wikia/CuratedContent/CuratedContentController.class.php [hunk 2]
```diff
@@ -440,14 +440,24 @@ public function getData( ) {
 
 		if ( !empty( $wgWikiaCuratedContent ) && is_array( $wgWikiaCuratedContent )  ) {
 			foreach ( $wgWikiaCuratedContent as $section ) {
-				// sections
-				if ( !empty( $section['title'] ) && empty( $section['featured'] ) ) {
+				// update information about node type
+				$section['node_type'] = 'section';
```
Comment (hakubo) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php:444]: hmm should we use camelCase?
since that is what we use on FE

Comment (rogatty) [inline extensions/wikia/CuratedContent/CuratedContentController.class.php:444]: All the rest of properties use underscore as they come from `$wgWikiaCuratedContent` like that. Should we convert all of them? We will have to revert the process when saving.

**Pull request merged**